uber/site_sections/preregistration.py

```diff
@@ -21,7 +21,9 @@ def check_dealer(group):
     elif not group.website:
         return "Please enter your business' website address"
     elif not group.description:
```

Comment:
Side note: we definitely need to extend the ``@validation`` functionality to let you mark whether or not something is a prereg-only validation.  There are a bunch of validations we only perform on the main prereg form and nowhere else, which is why these are here instead of in ``model_checks.py``.  However, now that we allow plugins to add to and override existing validations, we should allow them to do the same thing for these kinds of prereg-only validations.  (Theoretically plugins CAN do that now, it's just messier and we don't provide an official API for it.)